File: Stepik Course/Task2.py
from selenium import webdriver
import time

try:
    browser = webdriver.Chrome(r'C:\Users\kate_s\Downloads\chromedriver\chromedriver.exe')
    browser.get("http://suninjuly.github.io/find_xpath_form")

    input1 = browser.find_element_by_name("first_name")
    input1.send_keys("Kate")
    input2 = browser.find_element_by_name("last_name")
    input2.send_keys("Sem")
    input3 = browser.find_element_by_css_selector(".form-control.city")
    input3.send_keys("Kyiv")
    input4 = browser.find_element_by_id("country")
    input4.send_keys("Ukraine")
    button = browser.find_element_by_xpath(r'//button[@type = "submit"]')
    button.click()

finally:
    time.sleep(30)
    browser.quit()

File: Stepik course tasks in Java/Task2.java
package com.company;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

import java.util.List;
import java.util.concurrent.TimeUnit;


public class Main {

    public static void main(String[] args) throws InterruptedException {

        System.setProperty("webdriver.chrome.driver", "C:\\Users\\kate_s\\Downloads\\chromedriver\\chromedriver.exe"); //define path to the webdriver
        WebDriver driver = new ChromeDriver(); //   Create instance of ChromeDriver interface
        
    try {
        driver.get("http://suninjuly.github.io/find_xpath_form");

        WebElement f_name = driver.findElement(By.name("first_name"));
        f_name.sendKeys("Kate");
        WebElement l_name = driver.findElement(By.name("last_name"));
        l_name.sendKeys("Sam");
        WebElement city = driver.findElement(By.cssSelector(".form-control.city"));
        city.sendKeys("Kyiv");
        WebElement country = driver.findElement(By.id("country"));
        country.sendKeys("Ukraine");

        WebElement button = driver.findElement(By.xpath("//button[@type = \"submit\"]"));
        button.click();
    }
    
finally
        {
            TimeUnit.SECONDS.sleep(6);
            driver.quit();
        }
    }
}
